crates/ironrdp-cliprdr-format/README.md

```diff
@@ -2,6 +2,3 @@
 
 This Library provides the conversion logic between RDP-specific clipboard formats and
 widely used formats like PNG for images, plain string for HTML etc.
-
-### INVARIANTS
-- This crate expects the target machine's pointer size (usize) to be equal or greater than 32bits
```

Comment:
This restriction was not required, we were performing a checked conversion from usize to u32 anyway:
```rust
cast_int!("DIB header", "biImageSize", image_size).map_err(|_| BitmapError::InvalidSize)?,
```
(which I rewrote using `u32::try_from`, since the error is discarded anyway.)